inst/text/gitignore.txt

```diff
@@ -23,10 +37,43 @@ data/
 *.[rR][dD][aA][tT][aA]
 *.[rR][dD][sS]
 
-# Cross platform data files #
+# Cross-platform data files #
 *.[pP][aA][rR][qQ][uU][eE][tT]
 *.[fF][sS][tT]
 *.[qQ][sS]
 
 # MacOS folder attributes files #
 .DS_Store
+
+# produced vignettes
```

Comment:
Some of this is maybe specific to R packages while our gitignore file is more general purpose?
Is there a chance that somebody may want to e.g. have a vignettes folder for an R project that is not a package?

Reply:
Yeah, it's definitely specific to R package development rather than general analysis.

I can't think of why someone would have a vignettes directory outside of a package context but even so, this just ignores the built versions (pdf and HTML) and would still track any md / rmd files.

Reply:
Probably fine to keep in the gitignore - I guess having this as the 'default' is fine and people can amend the gitignore if needed for their own projects